src/test/java/DukeTest.java

```diff
@@ -0,0 +1,30 @@
+import duke.*;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+
+    Duke duke = new Duke("C:\\Users\\Rickie\\Documents\\University\\Year 2\\Semester 1\\CS2103T\\week 2\\ip\\Data\\taskList.txt");
```

Comment:
Consider using relative path instead of absolute path, incase project structure changes in the future